repos/tap/components/cmdOutput/renderOutput.js

```diff
@@ -1,25 +1,51 @@
-import React from "react"
+import React, { useMemo, useRef, useEffect } from "react"
 import { useStyle } from '@keg-hub/re-theme'
 import { Row } from '@keg-hub/keg-components/row'
 import { Icon } from '@keg-hub/keg-components/icon'
 import { View } from '@keg-hub/keg-components/view'
 import { Text } from '@keg-hub/keg-components/text'
-import { wordCaps, noOpObj } from '@keg-hub/jsutils'
-import { AnsiUp } from 'SVUtils/runner/ansiUp'
+import { wordCaps, noOpObj, mapObj } from '@keg-hub/jsutils'
 
-const ansiUp = new AnsiUp
-// ansiUp.ansi_to_html
+const useRunMessages = (messages) => useMemo(() => {
+  return mapObj(messages, (__, meta) => meta)
```

Comment:
seems like you don't really need `mapObj` anymore, `Object.values(messages)` would be simpler and clearer

Reply:
Ahh yah, good point. I'll update this